tests/nodes/3/config.yml

```diff
@@ -5,19 +5,8 @@ rpc:
   listening_addr: 127.0.0.1:41716
 
 cch:
-  ratio_btc_msat: 1
-  ratio_ckb_shannons: 5000
-  allow_expired_btc_invoice: true
-
-# Note that we are different in the sense we have 20 billions of CKB tokens in the genesis block.
-ckb_chain:
-  # sighash_all
-  funding_source_lock_script_code_hash: "0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8"
-  funding_source_lock_script_hash_type: "type"
-
-  # funding lock
-  funding_cell_lock_script_code_hash: "0x8090ce20be9976e2407511502acebf74ac1cfed10d7b35b7f33f56c9bd0daec6"
-  funding_cell_lock_script_hash_type: "type"
+  wrapped_btc_type_script_args: "0x32e555f3ff8e135cece1351a6a2971518392c1e30375c1e006ad0ce8eac07947"
+  lnd_cert_path: ../../../deploy/lnd-init/lnd-ingrid/tls.cert
 
```

Comment:
why nodes/1 and nodes/2 config.yml don't need this?
now in main, `config.yml` will ignored by git, they will be generated from `deployer/config.yml`.